scripts/local-universe.py

```diff
@@ -153,52 +158,27 @@ def handle_package(opts):
 
 def enumerate_dcos_packages(
         packages_path,
-        package_names,
+        packages,
         only_selected,
         dcos_version):
     """Enumarate all of the package and revision to include
 
     :param packages_path: the path to the root of the packages
     :type packages_path: pathlib.Path
-    :param package_names: list of package to include. empty list means all
-                         packages
-    :type package_names: [str]
+    :param packages: list of the name and version of packages to include. empty
+                     list means all packages
+    :type packages: [[str]]
```

Comment:
I know Python's tuples are just immutable lists, but being able to have a more specific type is useful.